examples/server_client.rs

```diff
@@ -3,18 +3,20 @@
 //! Note that in practice you don't want to implement a chat client using UDP.
 use std::io::stdin;
 
-use laminar::{config::NetworkConfig, error::Result, net::UdpSocket, Packet};
+use laminar::{Config, NetworkError, Packet, Socket, SocketEvent};
+use std::thread;
 
-const SERVER: &str = "localhost:12351";
+const SERVER: &str = "127.0.0.1:12351";
```

Comment:
Just out of curiosity, why the change from localhost?

Reply:
localhost I think requires to resolve and address can be converted to a `SocketAddr`.

Reply:
Was getting an error if I tried to bind to localhost.